CanvasColorSpaceProposal.md

```diff
@@ -72,6 +70,7 @@ partial dictionary CanvasRenderingContext2DSettings {
 partial dictionary WebGLContextAttributes {
   CanvasColorSpaceEnum colorSpace = "srgb";
   CanvasStorageFormatEnum storageFormat = "unorm8";
+  boolean srgbColorEncoding = false;
```

Comment:
I would rather say "unorm8Srgb cannot be used with canvas2d".
We would need to forbid srgbColorEncoding with any format other than unorm8 otherwise, which is the same class of restriction.

Reply:
The separate boolean has the advantages that
- it mirrors how alpha impacts the GL format for the backbuffer
- it doesn't add a format that cannot be used with 2D canvas